GTHC/web
# Add details to each shift modals

## Description

This PR does the following to the shift modals:
- Organizes the design of the Shift View Modal
- Added Time changing to both the Shift Create and Update Modals
- Updates User availability in the modals whenever time was changed

## Motivation and Context


This [time changing on update and create] was feature that has always been requested, and organizing the shift view modals was just mandatory as the shift data was just splashed across the modal.

## How Has This Been Tested?



1. Sign in and go to calendar.
2. Create a shift, and see that the time can be updated past what was selected by drag and drop.
3. Update a shift, and see again.
4. Click on a created shift and see the updated views.

## Screenshots (if appropriate):
**ShiftViewModal**
![image](https://user-images.githubusercontent.com/13587407/50550743-c7722700-0c43-11e9-8f6e-a1e6206c7d55.png)

**ShiftCreateModal**
![image](https://user-images.githubusercontent.com/13587407/50550762-1029e000-0c44-11e9-89d6-57284771ab65.png)



**ShiftUpdateModal**
![image](https://user-images.githubusercontent.com/13587407/50550759-030cf100-0c44-11e9-9fdc-80b26c3e78de.png)


## Types of changes


## Checklist:
